storage/src/test/scala/coop/rchain/storage/StorageTests.scala

```diff
@@ -0,0 +1,138 @@
+/*               __          __                                         *\
+**    __________/ /_  ____ _/_/___                                      **
+**   / ___/ ___/ __ \/ __ `/ / __ \     RChain API                      **
+**  / /  / /__/ / / / /_/ / / / / /     (c) http://rchain.coop          **
+** /_/   \___/_/ /_/\____/_/_/ /_/                                      **
+\*                                                                      */
+
+package coop.rchain.Storage
+
+// TODO: add timings to tests (get and put)
+
+
+object StorageTests {
+
+  def tests(): Unit = {
+
+    println("StorageTests.tests() begin")
+
+    loadFiles()
+
+    putGetDupSort()
+
+    println("StorageTests.tests() end")
+  }
+
+  def putGetDupSort():Unit = {
+
+    println("putGetDupSort() begin")
+
+    val dirName:Option[String] = Some("storageGetDupSortDb")
+    val name:Option[String] = Some("storageGetDupSort")
+
+    val storConf = new StorageConfig()
+    storConf.isKeyToValues = true
+    storConf.isWritable = true
+    storConf.baseDir = Some(System.getProperty("user.dir"))
+    storConf.dirName = dirName
+    storConf.name = name
+    assert(storConf.isValid())
+
+    var storage = new Storage(storConf)
+    storage.deleteFiles()
+
+    try {
+      storage.put(1,2)
+      val getInt = storage.getInts(1)
+      assert(getInt.get(0) == 2)
+
+      storage.put(3,4)
+      storage.put(3,5)
+      val getInts = storage.getInts(3)
+      assert(getInts.get(0) == 4)
+      assert(getInts.get(1) == 5)
+
+      storage.put(10L,20L)
+      // val getInt = lmdb.getLongLongs(10L)
+      // assert(getInt.get(0) == 2)
+
+      storage.put("a", "b")
+      val getStr = storage.getStrings("a")
+      val str = getStr.get(0)
+      assert(str == "b")
+
+      storage.put("c", "d")
+      storage.put("c", "e")
+      val getStrs = storage.getStrings("c")
+      assert(getStrs.get(0) == "d")
+      assert(getStrs.get(1) == "e")
+    }
+    catch {
+      case e:Throwable => {
+        println("putGetDupSort(): " + e)
+      }
+    }
+    finally {
+      storage.close()
+      storage.deleteFiles()
+    }
+
+    println("putGetDupSort() end")
+  }
+
+
+  def loadFiles():Unit = {
+
+    println("loadFile() begin")
+
+    val storeFlat =
+      "/home/griff/Documents/storage/2017-10-19_2_lmdbjava/stores/storeFlat.txt"
```

Comment:
LIES! :)

Reply:
What do you think of:

    val storeFlat =
      "<root>/src/test/scala/coop/rchain/storage/stores/storeFlat.txt"
    val storeNested =
      "<root>/src/test/scala/coop/rchain/storage/stores/storeNested.txt"
    val storeRecursive =
      "<root>/src/test/scala/coop/rchain/storage/stores/storeRecursive.txt"

I guess I could write the code that figures out what <root> is.

Reply:
My last sentence got rendered in Markdown in an unexpected way.  Here is my second attempt:

I guess I could write the code that figures out what <<root>> is.

Reply:
Still didn't work.  Third attempt:

I guess I could write the code that figures out what "root", with less-than-symbol on the left and greater-than-symbol on the right, is.

Reply:
Heh. I think I see what you're going for.

Is this runnable via `sbt test`, yet? If so, then I'd put in relative paths (that can be `realpath`'d if necessary) that let it work from wherever `sbt test` runs.